pkg/restclient/snapshot.go

```diff
@@ -42,3 +47,19 @@ func (c *Client) SnapshotInfo(snapname string) (api.SnapGetResp, error) {
 	err := c.get(url, nil, http.StatusOK, &snap)
 	return snap, err
 }
+
+// SnapshotDelete will delete Gluster Snapshot and respective lv
+func (c *Client) SnapshotDelete(snapname string) error {
+	url := fmt.Sprintf("/v1/snapshot/%s", snapname)
+	err := c.del(url, nil, http.StatusOK, nil)
```

Comment:
this should be changed as
```
```
we should expect HTTP status code 204 for a delete operation